cvat/settings/development.py

```diff
@@ -19,14 +19,14 @@
 SENDFILE_BACKEND = "django_sendfile.backends.development"
 
 # Cross-Origin Resource Sharing settings for CVAT UI
-UI_SCHEME = os.environ.get("UI_SCHEME", "http")
-UI_HOST = os.environ.get("UI_HOST", "localhost")
-UI_PORT = os.environ.get("UI_PORT", 3000)
+CVAT_UI_SCHEME = os.environ.get("CVAT_UI_SCHEME", "http")
+CVAT_UI_HOST = os.environ.get("CVAT_UI_HOST", "localhost")
+CVAT_UI_PORT = os.environ.get("CVAT_UI_PORT", 3000)
```

Comment:
The environment variable renaming looks good to me, but I think the Python variables should keep their old name. We usually don't use the `CVAT_` prefix for setting names (since they all belong to CVAT).

Reply:
As these vars are development specific, it is not really convenient to have just `UI_HOST` and `UI_PORT` as they are quite abstract for shared environment. 

> We usually don't use the CVAT_ prefix for setting names

I would not say about "usual" here.

- CVAT_SERVERLESS
- CVAT_POSTGRES_PASSWORD
- CVAT_DATASET_CACHE_TTL
- CVAT_VERSION
- CVAT_NUCLIO_HOST

and many more may be found.

Reply:
Of these, only `CVAT_BASE_URL` and `CVAT_HOST` are setting names (and I'd argue they shouldn't be...).

Reply:
Oh, you told only about setting names
I though about env names